lib/active_support/cache/database_store/model.rb

```diff
@@ -12,12 +12,15 @@ def self.truncate!
 
         scope :fresh, -> { where(arel_table[:expires_at].gt(Time.zone.now)) }
         scope :expired, -> { where(arel_table[:expires_at].lteq(Time.zone.now)) }
+        scope :outdated, -> (date = 1.month.ago) { where(arel_table[:created_at].lt(date)) }
```

Comment:
I think this is misleading, as "outdated" is fairly subjective. I suggest calling this `created_before` which is less ambiguous